src/common/extension-telemetry-events.ts

```diff
@@ -94,6 +94,7 @@ export enum TelemetryEventSource {
     LaunchPadFastPass,
     LaunchPadAllTests,
     LaunchPadAssessment,
+    LaunchPadMediumPass,
```

Comment:
This was the enum I was thinking of earlier, where we want to make sure to move it to the end.